### DISCUSSION TITLE
awaken dont work

### DISCUSSION THREAD

**Discussion by SnusM** [SnosMe/awakened-poe-trade#1048] [Q&A]
When the program is active, no function works.

**Comment by SnosMe** [comment]
https://snosme.github.io/awakened-poe-trade/issues

**Reply by SnusM** [reply]
It didn't help, I already reinstalled, deleted logs, updated drivers (I play on Directx 11), the program just opens, the combination Ctrl + D, Shift + space do not respond

**Reply by SnusM** [reply]
maybe this is a problem in windows or antivirus?

**Reply by SnusM** [reply]
Hello ?

**Reply by SnosMe** [reply]
Try removing config

**Reply by SnusM** [reply]
It was also written in the instructions, I already did it, it didn't help (

**Reply by SnusM** [reply]
is there such a function to check the integrity?

**Reply by SnusM** [reply]
I try to run it every day but unsuccessfully

**Reply by SnusM** [reply]
I have been playing with the program since the delirium league, everything was fine, but before the league I cleaned the logs and it stopped working

**Reply by SnosMe** [reply]
Your best bet is to create new Windows user and test there

**Reply by SnusM** [reply]
I created a local user, doesn't help

**Reply by SnusM** [reply]
when I try to open configs through the taskbar, he writes to me that he does not see the roaming/awakened-poe-trade/apt-data folder

**Reply by SnusM** [reply]
maybe that 's the problem ?

**Reply by SnusM** [reply]
So there is a solution to this problem, I have suspicions that the problem is in the system itself
